tasks/ocp-services-reboot/masters.yml

```diff
@@ -0,0 +1,18 @@
+---
+# ensure gluster is healthy before stopping another node
+- include_role:
+    name: openshift_storage_glusterfs_cluster_health
+
+- name: OCP | Include Tasks for Unschedule and Drain
+  import_tasks: tasks/unschedule-and-drain.yml
+  delegate_to: "{{ openshift_oc_tools_host }}"
+
+- name: OCP | Update OS | Restart Master API and Controllers
+  include_tasks: tasks/restart_master_services.yml
+  when: inventory_hostname in groups['masters']
```

Comment:
I don't think we need this condition anymore since we are in the reboot/masters.yml and it is only called for masters.

Reply:
Made change.